modules/openapi-generator/src/main/resources/kotlin-vertx-server/README.mustache

```diff
@@ -0,0 +1,3 @@
+{{^hideGenerationTimestamp}}
+    Project generated on : {{generatedDate}}
+{{/hideGenerationTimestamp}}
```

Comment:
Can we create the README.mustache based on https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/kotlin-server/README.mustache to start with?